docs/Prowlarr/prowlarr-setup-proxy.md

```diff
@@ -12,13 +12,13 @@ Just run VPN on the apps that actually need it, like your torrent client, and pe
 
     This isn't the same as the tracker, the API/Website scraping happens on the actual site and some indexers don't allow VPN usage on their site.
 
-In this guide i will try to explain how to setup Prowlarr to use your VPN from your torrent client.
+In this guide, I will try to explain how to setup Prowlarr to use your VPN from your torrent client.
```

Comment:
I'm not sure, but I seem to recall we switched from an "I" to "We" in the guides going forward. If not my bad.

Reply:
We

Reply:
@nuxencs I changed it to a suggestion you can commit